apps/portal/src/content/docs/components/language-selector.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Language Dialog
+description: LanguageDialog component for selecting and saving language preferences.
+---
+
+The `LanguageDialog` component allows users to select and save their language preferences.
+
+import { DocsPage } from "@/components/docs-page";
+import { LanguageDialog } from "@harnessio/ui/components";
```

Comment:
No need to import this unless it's used in the MDX. The `ComponentExample` handles importing everything it needs to display anything from the `ui` package.
```suggestion
```

Reply:
Done.